### DISCUSSION TITLE
Play several notes at the same time that is not a Chord

### DISCUSSION THREAD

**Discussion by seanjmiller** [melanchall/drywetmidi#217] [Q&A] [answered]
How can I play several notes at the same time not using the PatternBuilder.Chord()?
All of the examples I've seen show adding Notes sequentially. Thank you very much.

**Comment by melanchall** [comment, marked as answer]
You can use `MoveToPreviousTime` after each `Note` method call:

```csharp
patternBuilder
    .Note(...)
    .MoveToPreviousTime()
    .Note(...)
    .MoveToPreviousTime()
    .Note(...)
    .MoveToPreviousTime();
```

All these notes will start at the same time.

**Reply by seanjmiller** [reply]
Works great! Thanks!
